### DISCUSSION TITLE
v0.2.2

### DISCUSSION THREAD

**Discussion by owen-reorg** [bnb-chain/opbnb#84] [Announcements]
This is the Fermat Hardfork release for opBNB Mainnet.
It will be activated at block height 9397477, expected to occur on November 28, 2023, at 6 AM UTC.

All mainnet nodes must upgrade to this release before the hardfork.

### User Facing Changes

- Support reading private keys from AWS Secret Manager for `op-node`, `op-batcher`, and `op-proposer`. Refer to PR #72 for additional information.

### Partial Changelog

- #72: feat: support AWS key manager

### Docker Images

- ghcr.io/bnb-chain/op-node:v0.2.2
- ghcr.io/bnb-chain/op-batcher:v0.2.2
- ghcr.io/bnb-chain/op-proposer:v0.2.2

### Full Changelog

https://github.com/bnb-chain/opbnb/compare/v0.2.1...v0.2.2

<hr /><em>This discussion was created from the release <a href='https://github.com/bnb-chain/opbnb/releases/tag/v0.2.2'>v0.2.2</a>.</em>

**Comment by m-e-m-f-i-s-19** [comment]
In case, node hasn't been upgraded to the specified version, what will happen? I did node upgrade after hardfork, should I start node setup from scratch?

**Reply by owen-reorg** [reply]
Your node will fork from the canonical chain and stop at the height after the fork number.
If you are familiar with the engine API, you can upgrade your node and then reset op-geth to the block before the fork height.
Otherwise, it is easier to start the node from scratch.
If you don't need historical data, you can enable the snap sync of op-geth, which will make starting from scratch much faster. Refer to this [manual](https://docs.bnbchain.org/opbnb-docs/docs/build-on-opbnb/geth-sync) for more details.

**Comment by leotese** [comment]
Really nice, i think this will be a great chain!
